app/src/main/kotlin/com/wire/android/ui/calling/controlButtons/MicrophoneButton.kt

```diff
@@ -16,15 +16,24 @@ import com.wire.android.R
 import com.wire.android.ui.theme.wireDimensions
 
 @Composable
-fun MicrophoneButton() {
-    var isCallMuted by remember { mutableStateOf(true) }
+fun MicrophoneButton(
+    isCurrentCallMuted: Boolean,
+    onMuteCall: () -> Unit,
+    onUnMuteCall: () -> Unit
+) {
+    var isMuted by remember { mutableStateOf(isCurrentCallMuted) }
```

Comment:
Maybe this should be kept in the ViewModel and passed as a screen state rather than in the button itself, the button could be stateless and just use `isCurrentCallMuted`, what do you think?

Reply:
Agree, done 👍